tests/next_tests/integration_tests/multi_feature_tests/iterator_tests/test_anton_toy.py

```diff
@@ -68,8 +68,11 @@ def naive_lap(inp):
     for i in range(1, shape[0] + 1):
         for j in range(1, shape[1] + 1):
             for k in range(0, shape[2]):
-                out[i - 1, j - 1, k] = -4 * inp[i, j, k] + (
-                    inp[i + 1, j, k] + inp[i - 1, j, k] + inp[i, j + 1, k] + inp[i, j - 1, k]
+                out[i - 1, j - 1, k] = -4 * inp.asnumpy()[i, j, k] + (
```

Comment:
Couldn't we add an alias to the input numpy array outside of the loop nest and then use the alias inside?
```
inp_data = inp.asnumpy()
for ...
   for ...
      for ...
         out ... = inp_data[...]
```